openstack/rms/v1/policyassignments/requests.go

```diff
@@ -0,0 +1,163 @@
+		return nil, err
+	}
+
+	var r Assignment
+	_, err = c.Put(rootURL(c, domainId), b, &r, &golangsdk.RequestOpts{
+		MoreHeaders: requestOpts.MoreHeaders,
+	})
+	return &r, err
+}
+
+// Get is a method to obtain the assignment detail by its ID.
+func Get(c *golangsdk.ServiceClient, domainId, assignmentId string) (*Assignment, error) {
+	var r Assignment
+	_, err := c.Get(resourceURL(c, domainId, assignmentId), &r, &golangsdk.RequestOpts{
+		MoreHeaders: requestOpts.MoreHeaders,
+	})
+	return &r, err
+}
+
+// UpdateOpts is the structure required by the Update method to update the assignment configuration.
+type UpdateOpts struct {
+	// The name of the policy assignment.
+	Name string `json:"name" required:"true"`
+	// The description of the policy assignment.
+	Description *string `json:"description,omitempty"`
+	// The type of the policy assignment.
+	// The valid values are as follow:
+	// + builtin
+	// + custom
+	Type string `json:"policy_assignment_type,omitempty"`
+	// The period of the policy rule check.
+	Period string `json:"period,omitempty"`
+	// The configuration used to filter resources.
+	PolicyFilter PolicyFilter `json:"policy_filter,omitempty"`
+	// The ID of the policy definition.
+	PolicyDefinitionId string `json:"policy_definition_id,omitempty"`
+	// The configuration of the custom policy.
+	CustomPolicy *CustomPolicy `json:"custom_policy,omitempty"`
+	// The rule definition of the policy assignment.
+	Parameters map[string]PolicyParameterValue `json:"parameters,omitempty"`
+}
+
+// Update is a method to update the assignment configuration.
+func Update(c *golangsdk.ServiceClient, domainId, assignmentId string, opts UpdateOpts) (*Assignment, error) {
+	b, err := golangsdk.BuildRequestBody(opts, "")
+	if err != nil {
+		return nil, err
+	}
+
+	var r Assignment
+	_, err = c.Put(resourceURL(c, domainId, assignmentId), b, &r, &golangsdk.RequestOpts{
+		MoreHeaders: requestOpts.MoreHeaders,
+	})
+	return &r, err
+}
+
+// Enable is a method to enable the resource function of the policy assignment.
+func Enable(c *golangsdk.ServiceClient, domainId, assignmentId string) error {
+	_, err := c.Post(enableURL(c, domainId, assignmentId), nil, nil, &golangsdk.RequestOpts{
+		MoreHeaders: requestOpts.MoreHeaders,
+	})
+	return err
+}
+
+// Disable is a method to disable the resource function of the policy assignment.
+func Disable(c *golangsdk.ServiceClient, domainId, assignmentId string) error {
+	_, err := c.Post(disableURL(c, domainId, assignmentId), nil, nil, &golangsdk.RequestOpts{
+		MoreHeaders: requestOpts.MoreHeaders,
+	})
+	return err
+}
+
+// Delete is a method to remove the specified policy assignment using its ID.
+func Delete(c *golangsdk.ServiceClient, domainId, assignmentId string) error {
+	_, err := c.Delete(resourceURL(c, domainId, assignmentId), &golangsdk.RequestOpts{
+		MoreHeaders: requestOpts.MoreHeaders,
+	})
+	return err
+}
+
+// List is a method to query the list of the policy defunitions using given parameters.
```

Comment:
```suggestion
// List is a method to query the list of the buildin policy definitions using given parameters.
```
defunitions or definitions ?

Reply:
done.
built-in policy definitions